atc/api/ccserver/get.go

```diff
@@ -1,14 +1,14 @@
 package ccserver
 
 import (
+	"code.cloudfoundry.org/lager"
```

Comment:
should be pointing to v3
```suggestion
	"code.cloudfoundry.org/lager/v3"
```